src/main/java/com/google/api/generator/gapic/composer/rest/ServiceStubSettingsClassComposer.java

```diff
@@ -0,0 +1,172 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.composer.rest;
+
+import com.google.api.gax.httpjson.GaxHttpJsonProperties;
+import com.google.api.gax.httpjson.HttpJsonTransportChannel;
+import com.google.api.gax.httpjson.InstantiatingHttpJsonChannelProvider;
+import com.google.api.gax.rpc.ApiClientHeaderProvider;
+import com.google.api.generator.engine.ast.AnnotationNode;
+import com.google.api.generator.engine.ast.ConcreteReference;
+import com.google.api.generator.engine.ast.MethodDefinition;
+import com.google.api.generator.engine.ast.MethodInvocationExpr;
+import com.google.api.generator.engine.ast.ScopeNode;
+import com.google.api.generator.engine.ast.StringObjectValue;
+import com.google.api.generator.engine.ast.TypeNode;
+import com.google.api.generator.engine.ast.ValueExpr;
+import com.google.api.generator.engine.ast.Variable;
+import com.google.api.generator.engine.ast.VariableExpr;
+import com.google.api.generator.gapic.composer.common.AbstractServiceStubSettingsClassComposer;
+import com.google.api.generator.gapic.composer.comment.SettingsCommentComposer;
+import com.google.api.generator.gapic.composer.store.TypeStore;
+import com.google.api.generator.gapic.composer.utils.ClassNames;
+import com.google.api.generator.gapic.model.Service;
+import java.util.Arrays;
+
+public class ServiceStubSettingsClassComposer extends
+    AbstractServiceStubSettingsClassComposer {
+  private static final ServiceStubSettingsClassComposer INSTANCE =
+      new ServiceStubSettingsClassComposer();
+
+  protected static final TypeStore FIXED_REST_TYPESTORE = createStaticTypes();
```

Comment:
iT is already different one, isn't it? The superclass one is called `FIXED_TYPESTORE`, while this one is called `FIXED_REST_TYPESTORE`

Reply:
Can we make this private then?

Reply:
Please see the other comment about the same ting. I made it protected for consistency with the parent class. If you have strong preference, I can make it private.

Reply:
Yes, I would prefer this to be privately-scoped, since it's meant to be used just in this class (so it is not consistent with `FIXED_TYPESTORE` by nature).

Reply:
By replacing it everywhere else, I actually did not replace it in the original location, from which it all started. Classical one-off error in my find/replace =) Corrected